docs/reference/src/main/asciidoc/developmentmode.asciidoc

```diff
@@ -70,13 +69,11 @@ You should see the following log message during initialization of your applicati
 [[probe]]
 ==== Probe
 
-This tool allows to inspect the application CDI components at runtime. See also the http://probe-weld.itos.redhat.com/weld-numberguess/weld-probe[demo application hosted on OpenShift].
-
-JSON data are available through the REST API, eventually (if <<config-dev-mode,JMX support is enabled>>) through the MXBean of name `org.jboss.weld.probe:type=JsonData,context=ID` where ID should be replaced with an idenfitier of an application.
-
-However, a default UI - HTML client (single-page application) is only available in a web application at `{webappContextPath}/weld-probe`, e.g. `http://localhost:8080/weld-numberguess/weld-probe`.
-
-Right now, the integration is provided for WildFly 10, Tomcat and Jetty (Weld Servlet) and Weld SE.
+This tool allows you to inspect the application CDI components at runtime.
+There is a default UI - HTML client (single-page application), which is only available in web applications.
+Just point your browser to `port://host/webappContextPath/weld-probe`, e.g. `http://localhost:8080/weld-numberguess/weld-probe`.
```

Comment:
`port://host`?
You mean `protocol://host:port`?

Reply:
Oops, yes of course. Good catch!